src/hera.cpp

```diff
@@ -73,6 +74,7 @@ const map<string, hera_evm1mode> evm1mode_options {
   { "reject", hera_evm1mode::reject },
   { "fallback", hera_evm1mode::fallback },
   { "evm2wasm", hera_evm1mode::evm2wasm_contract },
+  { "runevm", hera_evm1mode::runevm_contract },
```

Comment:
Shouldn't need a new entry, it follows the same external API as evm2wasm. Just use it as "evm2wasm".

Reply:
Can't get the `evm2wasm` mode to work. `callSystemContract` returns empty, as the address has no code associated with it (https://github.com/ethereum/aleth/blob/master/libethereum/Executive.cpp#L342). I expected aleth's evmc to redirect the call to hera again. Here's the command I use:

```
./testeth -t GeneralStateTests/stExample -- --singlenet Byzantium --singletest "add11" --vm ~/dev/ewasm/hera/build/src/libhera.so --evmc evm1mode=evm2wasm --evmc sys:evm2wasm=runevm2.wasm --verbosity 2
```

Reply:
Ah you are correct. We had this issue for it: https://github.com/ethereum/evmc/issues/53

I think we may be better just merging your `runevm` function, but later we could generalise it to have "run locally" vs. "run through the state".